documentation/modules/proc-creating-kafka-cluster.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-kafka-cluster-{context}']
+= Creating a cluster
+
+Create a Kafka cluster, then a topic within the cluster.
+
+When you create a cluster, the Cluster Operator you deployed listens for new Kafka resources.
+
+.Prerequisites
+
+* For the Kafka cluster, a Cluster Operator is deployed
+* For the topic, a running Kafka cluster
+
+.Procedure
+
+. Log in to the Kubernetes cluster as a non-privileged user.
+
+. Create a new `my-cluster` Kafka cluster with 1 Zookeeper and 1 broker nodes.
++
+* Use `persistent-claim` storage
+* Expose the Kafka cluster outside of the Kubernetes cluster using an external listener configured to use a `nodeport`.
++
+--
+[source,yaml,subs=attributes+]
+----
+cat << EOF | kubectl create -n my-kafka-project -f -
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: 1
+    listeners:
+      plain: {}
+      tls: {}
+      external:
+        type: nodeport
+        tls: false
+    storage:
+      type: jbod
+      volumes:
+      - id: 0
+        type: persistent-claim
+        size: 100Gi
+        deleteClaim: false
+    config:
+      offsets.topic.replication.factor: 1
+      transaction.state.log.replication.factor: 1
+      transaction.state.log.min.isr: 1
+  zookeeper:
+    replicas: 1
+    storage:
+      type: persistent-claim
+      size: 100Gi
+      deleteClaim: false
+  entityOperator:
+    topicOperator: {}
+    userOperator: {}
+EOF
+----
+--
+
+. Verify that the cluster was successfully deployed:
```

Comment:
"Verify" suggests that reader has to actively check something. Really they're just waiting.
```suggestion
. Wait for the cluster to be deployed:
```